### PR TITLE
Add a clarifying comment to autostart.rb

### PR DESCRIPTION
**Pull request by benzrf**

### CODE REVIEW

File: lib/celluloid/autostart.rb [hunk 1]
```diff
@@ -1,5 +1,6 @@
 require "celluloid"
 
+# start up the basic Celluloid services - only necessary to do this manually in old versions
```
Comment (tarcieri) [inline lib/celluloid/autostart.rb]: `only necessary to do this manually in old versions` not sure what you mean by that

Comment (benzrf) [inline lib/celluloid/autostart.rb]: do you mean that the phrasing is unclear, or are you asking me to document exactly which version it changed at?

Comment (tarcieri) [inline lib/celluloid/autostart.rb]: Both would be nice: `celluloid/autostart` was the preferred way to call this for quite some time, so some additional clarification on the history would be good.

Comment (digitalextremist) [inline lib/celluloid/autostart.rb]: The line in question can also be wrapped in `if $CELLULOID_BACKPORTED`

**Comment by tarcieri** [comment]
Closing as stale

**Pull request closed without merging**